plat/qti/docs/LICENSE.qcom.txt

```diff
@@ -0,0 +1,206 @@
+PLEASE READ THIS LICENSE AGREEMENT ("AGREEMENT") CAREFULLY.  THIS AGREEMENT IS
```

Comment:
The license that you use must be BSD. Alternatively, you can have the code dual licensed if one of the two licenses is BSD. For example:
```
SPDX-License-Identifier: BSD-3-Clause OR MIT
```